src/allmydata/test/test_istorageserver.py

```diff
@@ -17,18 +19,31 @@
     # fmt: off
     from future.builtins import filter, map, zip, ascii, chr, hex, input, next, oct, open, pow, round, super, bytes, dict, list, object, range, str, max, min  # noqa: F401
     # fmt: on
+else:
+    from typing import Set
 
 from random import Random
+from unittest import SkipTest
 
 from twisted.internet.defer import inlineCallbacks, returnValue
 from twisted.internet.task import Clock
+from twisted.internet import reactor
+from twisted.web.server import Site
+from twisted.web.client import HTTPConnectionPool
+from hyperlink import DecodedURL
+from treq.api import set_global_pool as set_treq_pool
```

Comment:
Even though this looks like a public API, it has no documentation (in source or on readthedocs) and it is below a giant "Private API" comment in the implementation. :(

What does it even do?  Can we really use it?

Reply:
I need some way to disable persistent connections, but that can be done a different way, probably.